src/Options.cc

```diff
@@ -554,7 +553,10 @@ Options parse_cmdline(int argc, char** argv) {
                 break;
 
             case '?':
-            default: usage(zargs[0], 1); break;
+            default:
+                fprintf(stderr, "ERROR: Unrecognized argument %s\n", zargs[optind - 1]);
+                usage(zargs[0], 1);
```

Comment:
```suggestion
                exit(1);
```

I'm not a fan of the `usage()` spill personally either (slightly related): https://github.com/zeek/zeek/commit/55655f5ea11e48d5de2696ff49f5577849465b4e).

How about suppressing usage (inspiration from grep)?
```
$ grep -Y | grep more
grep: invalid option -- 'Y'
Usage: grep [OPTION]... PATTERNS [FILE]...   // think we can skip.
Try 'grep --help' for more information.
```

And possibly also `zeek -r` or other cases with missing arguments if it's easy:
```
$ grep -C
grep: option requires an argument -- 'C'
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
```

This might break some people's workflows/tests, but :man_shrugging:

Reply:
I changed it to look like this:

```
$ zeek --aoeu
ERROR: Unrecognized option --aoeu
Try 'zeek --help' for more information.
```

and:

```
$ zeek -r
ERROR: Option -r requires an argument.
Try 'zeek --help' for more information.
```

thoughts?